src/main/scala/io/github/evis/scalafix/maven/plugin/params/CompiledDirectoryParam.scala

```diff
@@ -1,10 +1,11 @@
 package io.github.evis.scalafix.maven.plugin.params
 
-import java.nio.file.Paths
-
 object CompiledDirectoryParam {
 
-  def apply(compiledDirectory: String): MojoParam = {
-    _.withClasspath(List(Paths.get(compiledDirectory)).filter(_.toFile.exists))
+  def apply(dirs: Iterable[String]): MojoParam = {
+    _.withClasspath(dirs.map(getPath).filter(_.toFile.exists).toList)
```

Comment:
Seems we should get rid of duplicates here too: either by making `dirs` a Set (I'd prefer it), or by calling `distinct`.